s3/s3_test.go

```diff
@@ -0,0 +1,95 @@
+package s3
+
+import (
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/spf13/afero/test"
+)
+
+var s3bucket string
+
+func init() {
```

Comment:
Arguably, storage is really really cheap these days, 0.0245 per gb for standard storage.